packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -55,6 +55,11 @@ for PACKAGE in ${PACKAGES[@]}; do
         # sync queues
         echo "Migrating the database"
         yarn migrate
+
+        # After running migrations it's good to ensure that the analytics table is fully built
+        echo "Building analytics table"
+        yarn workspace @tupaia/data-api install-mv-refresh
+        yarn workspace @tupaia/data-api build-analytics-table
```

Comment:
This only builds the analytics table if it doesn't already exist, otherwise it skips it